examples/projects/VS2015/autobahn-cpp-examples.vs2015.sln

```diff
@@ -0,0 +1,84 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I agree. The amount of build file insanity for VS is plain ridiculous. cmake should be used, not native VS build crap.